guidelines/index.html

```diff
@@ -13,7 +13,7 @@
     </head>
     <body>
         <section id="abstract">
-            <p>Web Content Accessibility Guidelines (WCAG) 2.2 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content more accessible to a wider range of people with disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement, speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines address accessibility of web content on desktops, laptops, tablets, and mobile devices. Following these guidelines will also often make Web content more usable to users in general.</p>
+            <p>Web Content Accessibility Guidelines (WCAG) 2.2 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content more accessible to a wider range of people with disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement, speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines address accessibility of web content on any kind of device (including desktops, laptops, kiosks, and mobile devices). Following these guidelines will also often make Web content more usable to users in general.</p>
```

Comment:
I don't think this needs to be in brackets:

```suggestion
            <p>Web Content Accessibility Guidelines (WCAG) 2.2 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content more accessible to a wider range of people with disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement, speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines address accessibility of web content on any kind of device including desktops, laptops, kiosks, and mobile devices. Following these guidelines will also often make Web content more usable to users in general.</p>
```

Reply:
I think there needs to be something after the "any kind of device" to give that appropriate emphasis. If it wasn't a bracket, I think we'd end up with: "on any kind of device; including desktops", which is slightly less obvious. I don't have a strong opinion either way, it doesn't seem like a big issue.

Reply:
at the very least i think a comma would be good, or a dash. but i can live with no separator at all.

I'm actually more concerned now, re-reading this, with the awkward `; but will not ...` earlier in that paragraph. it reads super clunky.

Reply:
@WilcoFiers the intent was to make it clearer that the guidelines cover web content on any device, hence the use of the parentheses to demarcate that a bit more clearly and emphasize that the parenthetical items were examples, not an exhaustive list (which was one of the concerns in some of the discussions).

Are you arguing that the use of parentheses actually changes the meaning, as opposed to the use of commas? If not, since the current wording was adopted through a full review by the working group last spring, I'm inclined to leave it as is.

Reply:
> at the very least i think a comma would be good, or a dash. but i can live with no separator at all.
> 
> I'm actually more concerned now, re-reading this, with the awkward `; but will not ...` earlier in that paragraph. it reads super clunky.

@patrickhlauke this is pre-existing language and was not raised as part of the original issue. I also think it's not a great use of a semicolon, but that is not part of the change in this PR, and so is scope creep.